src/components/combobox/combobox.tsx

```diff
@@ -257,13 +257,13 @@ export class Combobox implements LabelableComponent, FormComponent, InteractiveC
    * Fired when the combobox is opened
    * @internal
    */
-  @Event() calciteComboboxOpen: EventEmitter;
+  @Event() calciteInternalComboboxOpen: EventEmitter;
```

Comment:
https://github.com/Esri/calcite-components/issues/1688

Reply:
Should we be removing @internal from Open/Close events then and making them public? Or did you mean specifically for combobox?

Reply:
I concur with @driskull on this one too.  Open/Close events should probably be publicly exposed.  Especially since this is a top-level component that isn't emitting this event for another Calcite component to listen for.  In that case, just removing the `@internal` label on it and restoring the original event name should suffice.

Reply:
I'm just questioning why these are internal. We may want to expose them at some point but we can do so later if needed.

Reply:
Should there be a follow-up issue for all top-level components to publicly expose Open/Close events, or does this need to be discussed for all components, including nested?

Reply:
I like the idea of starting a pattern discussion on open/close events and if places that use them should expose them publicly if they aren't already.  Let's create a discussion for this here: https://github.com/Esri/calcite-components/discussions